src/oscar/apps/catalogue/abstract_models.py

```diff
@@ -77,6 +77,16 @@ def has_attributes(self):
         return self.attributes.exists()
 
 
+class CategoryQuerySet(MP_NodeQuerySet):
+    def public(self):
```

Comment:
Can we change this to `browsable()` to be consistent with `ProductQuerySet`, and add a docstring to this method?

Reply:
Thanks for the feedback. I've renamed `public` to `browsable`.